cli/cmd/dashboard.go

```diff
@@ -10,40 +10,42 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	proxyPort = -1
-)
+var dashboardProxyPort int
+var dashboardSkipBrowser bool
 
 var dashboardCmd = &cobra.Command{
 	Use:   "dashboard [flags]",
 	Short: "Open the Conduit dashboard in a web browser",
-	Long:  "Open the Conduit dashboard in a web browser.",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if proxyPort < 0 {
-			return fmt.Errorf("port must be greater than or equal to zero, was %d", proxyPort)
+		if dashboardProxyPort < 0 {
+			return fmt.Errorf("port must be greater than or equal to zero, was %d", dashboardProxyPort)
 		}
 
-		kp, err := k8s.InitK8sProxy(shell.NewUnixShell().HomeDir(), kubeconfigPath, proxyPort)
+		kubernetesProxy, err := k8s.InitK8sProxy(shell.NewUnixShell().HomeDir(), kubeconfigPath, dashboardProxyPort)
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "Failed to initialize proxy: %s", err)
 			os.Exit(1)
 		}
 
-		url, err := kp.URLFor(controlPlaneNamespace, "/services/web:http/proxy/")
+		url, err := kubernetesProxy.URLFor(controlPlaneNamespace, "/services/web:http/proxy/")
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "Failed to generate URL for dashboard: %s", err)
 			os.Exit(1)
 		}
 
-		fmt.Printf("Opening [%s] in the default browser\n", url)
-		err = browser.OpenURL(url.String())
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to open URL %s in the default browser: %s", url, err)
-			os.Exit(1)
+		fmt.Printf("Conduit dashboard available at:\n%s\n", url.String())
+
+		if !dashboardSkipBrowser {
+			fmt.Printf("Opening [%s] in the default browser\n", url)
```

Comment:
maybe we don't need this extra print statement anymore, as we're providing the URL above?